src/main/java/seedu/address/ui/FocusCard.java

```diff
@@ -37,21 +64,42 @@
     @FXML
     private Label email;
     @FXML
+    private Label address;
+    @FXML
+    private Label seniority;
+    @FXML
     private Label applicationStatus;
     @FXML
     private Label candidateStatus;
     @FXML
     private Label availability;
     @FXML
-    private FlowPane tags;
+    private FlowPane statusFocusPane;
+    @FXML
+    private FlowPane availableDaysFocus;
+    @FXML
+    private ImageView displayPicture;
     /**
      * Creates a {@code CandidateCode} with the given {@code Candidate} and index to display.
      */
-    public FocusCard(Candidate candidate, int displayedIndex) {
+
+    public FocusCard(Candidate candidate) throws FileNotFoundException {
         super(FXML);
+        requireNonNull(candidate);
         this.candidate = candidate;
+        id.setText(candidate.getStudentId().toString());
+        name.setText(candidate.getName().fullName);
+        phone.setText(candidate.getPhone().value);
+        email.setText(candidate.getEmail().value);
+        course.setText(candidate.getCourse().course + ", " + candidate.getSeniority().seniority);
+        displayPicture.setImage(new Image(new FileInputStream(BLANK_PICTURE_PATH)));
+        setApplicationStatus(candidate.getApplicationStatus());
+        setInterviewStatus(candidate.getInterviewStatus());
+        setAvailableDays(candidate.getAvailability());
```

Comment:
If we are adding the confirmed interview slot in the future to the `FocusCard`, can link that future PR to #70 !